pkg/apiserver/webhooks/datavolume-validate.go

```diff
@@ -70,6 +70,19 @@ func validateDataVolumeName(name string) []metav1.StatusCause {
 	return causes
 }
 
+func validateContentTypes(sourcePVC *v1.PersistentVolumeClaim, spec *cdiv1.DataVolumeSpec) (bool, cdiv1.DataVolumeContentType, cdiv1.DataVolumeContentType) {
+	sourceContentType := cdiv1.DataVolumeKubeVirt
+	contentType, found := sourcePVC.Annotations[controller.AnnContentType]
+	if found {
```

Comment:
Is it really the preferable golang style? looks a bit ugly to me
https://stackoverflow.com/questions/26545883/how-to-do-one-liner-if-else-statement

Reply:
That question is about an if then else type ternary operator. In this case my main goal is to keep sourceContentType variable outside of the scope of the function, since it is only used inside the if statement. But it is more a style preference than a hard requirement.

Reply:
done